src/app/column-chart/column-chart.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-column-chart',
+  templateUrl: './column-chart.component.html',
+  styleUrls: ['./column-chart.component.css']
+})
+
+export class ColumnChartComponent implements OnInit {
+  options1: Object;
+  options2: Object;
```

Comment:
Also it is a good practice to use exact types instead of Object
You can create typescript interface and declare all properties there and use this interface here

Reply:
Done